mxcubeweb/core/components/queue.py

```diff
@@ -2078,6 +2080,20 @@ def init_queue_settings(self):
             "auto_add_diff_plan", False
         )
 
+        self.app.REMEMBER_PARAMETERS_BETWEEN_SAMPLES = (
+            HWR.beamline.queue_manager.get_property(
+                "default_remember_parameters_between_samples", False
```

Comment:
Looks good in general, do we need to call it `default_remember_parameters_between_samples` with the `default` prefix, would not `remember_parameters_between_samples` be sufficient ? I guess one understands that the user can change the option ?

Reply:
Both can work. I think that providing the `default` prefix is a good idea for fields that may be modified by the user, to avoid some awkward bugs with e.g., some fields not resetting - but that's only me preference :)

Reply:
There are two levels of "default" here, aren't there?

1. `False` is the default fall back value if `default_remember_parameters_between_samples` is not set, right?
2. and then `default_remember_parameters_between_samples` is also a "default" because it can be overridden via the checkbox in the UI, correct? It is the default value when starting MXCuBE.

Reply:
:), yes can be a bit confusing. I would not make things to complicated. I don't think we need the have the fallback/default argument configurable (`False` in this case) . I also don't think we need to keep the `default` prefix, some parameters are changeable by the users other are not, do we need to distinguish between them ? In that case a comment is perhaps more suitable.

Reply:
Okay, maybe too many defaults there :D will remove the prefix then.